src/lidar/read_rp_lidar.cpp

```diff
@@ -12,60 +12,84 @@
 
 using namespace navtypes;
 
-std::atomic<bool> lidar_initialized(false);
 constexpr uint32_t RPLIDAR_BAUDRATE = 115200;
+const double MM_TO_M = 1000;
+
+std::atomic<bool> lidar_initialized(false);
+std::atomic<datatime_t> lidar_time(datatime_t{});
```

Comment:
As mentioned below, `lidar_initialized` doesn't need to be atomic, and `lidar_time` isn't needed. Read the comments below for reasoning and shit